### PR TITLE
[Remove Stream usage:document-library-test]LPS-176227 Remove Stream usages in method body from DLFileEntryIndexe…

### PR DESCRIPTION
**Pull request by slnn**
https://issues.liferay.com/browse/LPS-176227

### CODE REVIEW

File: modules/apps/document-library/document-library-test/src/testIntegration/java/com/liferay/document/library/search/test/DLFileEntryIndexerLocalizedContentTest.java [hunk 1]
```diff
@@ -92,18 +91,9 @@ public void testJapaneseContent() throws Exception {
...
-		Stream.of(
-			word1, word2
-		).forEach(
-			searchTerm -> {
-				Document document = _search(searchTerm, LocaleUtil.JAPAN);
-
-				assertLocalization(contentStrings, document);
-			}
-		);
+		assertLocalization(contentStrings, _search("作成", LocaleUtil.JAPAN));
```
Comment (dantewang) [inline modules/apps/document-library/document-library-test/src/testIntegration/java/com/liferay/document/library/search/test/DLFileEntryIndexerLocalizedContentTest.java:96]: @slnn Can you check if SF complains when you remove the empty above this line? If not, please remove the empty line here and below.

**Comment by slnn** [comment]
ci:stop

**Comment by slnn** [comment]
ci:test:sf

**Comment by slnn** [comment]
ci:test:relevant

**Comment by catarineaguiar** [comment]
ci:test:document

**Comment by catarineaguiar** [comment]
no regressions found.

**Comment by dantewang** [comment]
ci:forward

cc @shuyangzhou @tinatian

**Pull request closed without merging**